frontend/src/components/widgets/inputs/FontInput.vue

```diff
@@ -101,17 +102,42 @@ export default defineComponent({
 	data() {
 		return {
 			open: false,
-			minWidth: 0,
-			entries: [] as SectionsOfMenuListEntries,
-			activeEntry: undefined as undefined | MenuListEntry,
+			entries: [] as MenuListEntry[],
+			activeEntry: undefined as MenuListEntry | undefined,
+			highlighted: undefined as MenuListEntry | undefined,
+			entriesStart: 0,
+			minWidth: this.isStyle ? 0 : 300,
 		};
 	},
 	async mounted() {
-		const { entries, activeEntry } = await this.updateEntries();
-		this.entries = entries;
-		this.activeEntry = activeEntry;
+		this.entries = await this.getEntries();
+		this.activeEntry = this.getActiveEntry(this.entries);
+		this.highlighted = this.activeEntry;
 	},
 	methods: {
+		floatingMenu() {
+			return this.$refs.floatingMenu as typeof FloatingMenu;
+		},
+		scroller() {
+			return ((this.$refs.menulist as typeof MenuList).$refs.scroller as typeof LayoutCol)?.$el as HTMLElement;
+		},
+		setOpen() {
+			this.open = true;
+			// Reset the highlighted entry to the active one
+			this.setHighlighted(this.activeEntry);
+			// Scroll to the active entry (the scroller div does not yet exist so we must wait for vue to render)
+			nextTick((): void => {
```

Comment:
`await nextTick();` is a slightly nicer syntax. (You'll need to prefix `setOpen() {` with `async`.)